sdks/python/apache_beam/io/gcp/datastore/v1/helper.py

```diff
@@ -295,6 +300,9 @@ def _next_batch(self):
     resp = self._datastore.run_query(self._req)
     return resp
 
+  def __next__(self):
+    return next(self.__iter__())
```

Comment:
This will always return the first element. Is this the expected behaviour?

Reply:
No this is indeed an error. This method is actually now also superfluous if I use your suggestion for the test in https://github.com/apache/beam/pull/5715#discussion_r199078083. I removed it